frame/support/test/tests/runtime_metadata.rs

```diff
@@ -0,0 +1,222 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2020-2022 Parity Technologies (UK) Ltd.
```

Comment:
```suggestion
// Copyright (C) Parity Technologies (UK) Ltd.
```

We removed copyright years.